airflow/Dockerfile

```diff
@@ -30,8 +31,10 @@ RUN pip3 install -r /opt/airflow/requirements.txt
 
 COPY --chown=airflow:0 ./opt/ /opt/airflow/
 COPY --from=1 /mocker /usr/lib/mocker
+COPY --chown=airflow:0 --from=etna-py /etna-py /etna-py
```

Comment:
So this is the trick as far as performance goes, and it's an unfortunate consequence of how python packaging works.

In ruby and node, we work around this by performing the 'install' on just the requirements files first -- so, the `Gemfile` or the `package.json`, THEN copying the actual source over.  This means that as long as the requirements files themselves do not change, docker rebuilds see the install steps as cached and do not re-execute them.

Unfortunately, python installs of packages actually perform a --hard copy-- on install, rather than linking to their source path, meaning that if the entire source code isn't installed at the time of `pip install`, then the source code is not copied into the final virtualenv location.

If we wanted to improve the performance here, essentially what we'd want to do is

```
COPY --chown=airflow:0 --from=etna-py /etna-py/{setup.py,... project skeleton files only} /etna-py
RUN pip install /etna-py
COPY --chown=airflow:0 --from=etna-py /etna-py (actual installed destination of the etna-py library)
```

In that case, the first two steps involved with 'installing' are executed only when the actual skeleton files (usually for new dependencies) change, while the source code is a mere copy on delivery situation.  But that involves some magic similar to the `link-etnapy` and `link-magby` development steps to determine the correct locations.